README.md

```diff
@@ -53,8 +53,7 @@ Although we will likely lint with `flake` so don't worry too much about it.
 
 The `tests` folder contains unit tests. These can be run manually by using the command:
 
```

Comment:
Somewhere in the README there should be a line that states that `bmdal` is a submodule and must be initialised with `git submodule update --init --recursive`